src/Microsoft.Azure.SignalR.Common/Endpoints/AccessKey.cs

```diff
@@ -11,16 +11,20 @@ namespace Microsoft.Azure.SignalR
     internal class AccessKey
     {
         public string Id { get; protected set; }
```

Comment:
also remove set for `Id` and `Value`?

Reply:
The value for `Id` and `Value` in `AadAccessKey` (which is inherited from `AccessKey`) are updated from the `GetAccessKey` API results. So it needs to be settable and the level should be **protected**;